src/app/main/component/ubs/services/order.service.ts

```diff
@@ -17,8 +17,12 @@ export class OrderService {
   locationSubject = new Subject();
   locationSub = new Subject();
   currentAddress = new Subject();
+  public getStepperFinal$: Observable<boolean>;
+  public getStepperFinalSubject = new BehaviorSubject<boolean>(false);
```

Comment:
I don't really understand why you need both subject and observable at the same time
You should have only one source of truth but in your case, you have two properties for the same data
It could be the case when you have different values in both properties and it's impossible to realize which one we should use

Reply:
I try to do in in other way. But it doesn't work.

here is link-->

https://stackoverflow.com/questions/44066905/angular-2-send-data-from-component-to-service/44067238#44067238

Reply:
I'm still not sure about it, we should discuss offline

Reply:
I fix it already.

Reply:
Done